docs/source/message.rst

```diff
@@ -0,0 +1,27 @@
+Message
+=======
+
+A message is an object that was created to wrap the data from the components and add to it
+more necessary information for the developer to keep track of the data.
+With the messages, the components can send the data to the database they use and receive data as well.
+For this moment, all the components use Redis message broker.
+
+Every message contains:
+1. unique id
+2. payload - a frame or prediction to send or receive from the message broker.
+3. history - a documentation of when data came in and out of the component.
+
+Payloads:
+We have 2 payloads - frame and prediction.
```

Comment:
The frame payload can be address to the shared memory.

Can you add a link to the payload class?

Reply:
added